openapi.json

```diff
@@ -0,0 +1,185 @@
+{
+  "openapi": "3.0.1",
+  "info": {
+    "version": "9.0.0",
+    "title": "GitHub v3 REST API",
+    "license": {
+      "name": "MIT"
+    }
+  },
+  "servers": [
+    {
+      "url": "https://api.github.com"
+    }
+  ],
+  "paths": {
+    "/repos/:owner/:repo/issues": {
+      "get": {
+        "summary": "List issues for a repository",
+        "description": "...",
+        "operationId": "list-for-repo",
+        "tags": [
+          "issues"
+        ],
+        "externalDocs": {
+            "description": "API method documentation",
+            "url": "https://developer.github.com/v3/issues/#list-issues-for-a-repository"
+        },
+        "parameters": [
+          {
+            "name": "owner",
+            "in": "path",
+            "schema": {
+              "type": "string"
+            },
+            "required": true,
+            "description": ""
+          },
+          {
+            "name": "repo",
+            "in": "path",
+            "schema": {
+              "type": "string"
+            },
+            "required": true,
+            "description": ""
+          },
+          {
+            "name": "milestone",
+            "in": "query",
+            "schema": {
+              "oneOf": [
+                {
+                  "type": "integer",
+                  "format": "int32",
+                  "minimum": 1
+                },
+                {
+                  "type": "string",
+                  "enum": ["*"]
+                }
+              ]
```

Comment:
Getting the "oneOf" schema automatically from scraping the docs might be tricky, but if it’s useful we will find a way